benchmarks/benchmarks/analysis/distances.py

```
@@ -1,6 +1,12 @@
 import MDAnalysis
 import numpy as np
 
+
+try:
+    from MDAnalysis.lib import mdamath
+except:
```

Is there any realistic type of exception you're expecting here apart from `ImportError` for some old versions of MDAnalysis? In other words, why catch all exceptions here? It looks like, to be fair, I wrote some of the generic `except` into this file 4 years ago, but not sure that makes it a good idea.

---

Yeah I was just following you haha. Happy to take it out.